NEWS

```diff
@@ -11,9 +11,171 @@
+* Added option to enable using cross-device dmabuf file descriptor for rocm
+
+#### Java
+* Added Java bindings for exported memh feature
+
+#### Tests
+* Added a rocm docker container for testing
+* Added option to send client Id in IODEMO test
+* Added support for repeating connection to the same server in IODEMO test
+* Added receive callback for receiving FIN message in UCX examples
+
+#### Tools
+* Added user-side memcpy option for AM benchmarks in ucx_perftest
+
+#### Build
+* Added UCX LUA dissectors for 64-bit little-endian
+* Added a separate xpmem deb subpackage
+* Added ARM-based artifacts.
+* Removed dependency on libnuma
+
+### Bugfixes:
+
+#### UCP
+* Fixed crash during connection manager cleanup
+* Fixed AM send status handler in the new protocol infrastructure
+* Fixed rkey index calculation for Rendezvous protocol
+* Fixed rcache dump function
+* Removed logging from rkey unpack in release mode
+* Removed double resource cleanup from Rendezvous rkey protocol
+* Fixed rtr/ppl error flow
+* Added workaround to error handling of rndv/get/zcopy
+* Replay pending requests of wireup EP CM during connection establishment to prevent potential ordering issues and wrong configuration
+* Pass user-provided memory type to the function that checks whether the buffer can be sent inline or not
+* Removed test memory registration during UCP context initialization
+* Fixed CPU/device atomics selection in the new protocol infrastructure
+* Multiple fixes in the new protocol infrastructure information output
+
+#### UCT
+* Fixed exported memh packing
+* Fixed an error in checking return status of multi-threaded memory registration function
+* Introduced MD capability for non-blocking registration memory types
+
+#### RDMA CORE (IB, ROCE, etc.)
+* Added check for UAR support to memory domain opening
+* Fixed updating port counters for devx qp
+* Fixed ibv_create_cq error message on node without Infiniband
+* Fixed performance degradation due to using 2 paths on NDR400 by default
+* Removed unnecessary async lock which otherwise would block UD progress
+
+#### UCS
+* Fixed displaying wrong environment variable suggestions
+* Fixed VFS warning output
+* Fixed SEGV in ucs_debug_backtrace_next(), upon previous SEGV handling, due to ENOMEM situation
+* Fixed memory corruption when using UCX_MPOOL_FIFO=y
+
+#### UCM
+* Fixed mremap() override
+
+#### GPU (CUDA, ROCM)
+* Fixed usage of dmabuf when the buffer is not page-aligned
+* Removed async_cb from cuda_copy to avoid the issue with UCP worker async lock
+* Fixed device detection for host address for rocm
+
+#### Java
+* Fixed leakage of jucx_request global references
+
+#### Documentation
+* Updated ucp_worker_release_address description
+
+#### Tests
+* Disabled gtests on wireless interfaces
+* Disabled JAVA tests on docker interfaces
+* Fixed wrong usage of ep_close in examples
+* Fixed failures on tests built in single-treaded mode
+* Multiple fixes in unit tests
```

Comment:
maybe mention just `Fixed wrong usage of ep_close in examples`, as all the rest seems to be infra internals

Reply:
Updated.